### PR TITLE
Wrong variable name

### PR DESCRIPTION
**Pull request by nanaya**
Didn't pay enough attention.

### CODE REVIEW

File: app/Models/Forum/Topic.php [hunk 1]
```diff
@@ -80,8 +80,8 @@ public static function createNew($forum, $params, $pollParams = null)
             }
 
             if (($params['cover'] ?? null) !== null) {
```
Comment (vendethiel) [inline app/Models/Forum/Topic.php]: `isset($params['cover'])`?

Comment (nanaya) [inline app/Models/Forum/Topic.php]: it might be set with value of null.

Comment (vendethiel) [inline app/Models/Forum/Topic.php]: sure?

```
$ php -r '$a = [null]; var_dump(isset($a[0]));'
bool(false)
```

Comment (nanaya) [inline app/Models/Forum/Topic.php]: what the hell. Also, no.

Comment (nanaya) [inline app/Models/Forum/Topic.php]: I think I did that somewhere else but the fact I forgot it already means I shouldn't use it.

Comment (vendethiel) [inline app/Models/Forum/Topic.php]: okay, that's a fair point :P.

Comment (nanaya) [inline app/Models/Forum/Topic.php]: more like because it's not a function but a language construct?

Comment (vendethiel) [inline app/Models/Forum/Topic.php]: yes. That's why for the longest time, `isset(literal)` was a parse error. It now has a better error message now, though.

Comment (nanaya) [inline app/Models/Forum/Topic.php]: still parse error for null.
```
>>> isset(null)
PHP Parse error: Syntax error, unexpected ')', expecting '[' on line 1
```

Comment (nanaya) [inline app/Models/Forum/Topic.php]: and of course :D 

```
>>> isset('1')
PHP Parse error: Syntax error, unexpected ')', expecting T_PAAMAYIM_NEKUDOTAYIM on line 1
```

Comment (vendethiel) [inline app/Models/Forum/Topic.php]: what's your PHP version? as said, it's a very recent change

```
$ php -v
PHP 5.5.33 (cli) (built: Mar 25 2016 10:45:27)
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2015 Zend Technologies
$ php -r "isset(null);"
Fatal error: Cannot use isset() on the result of an expression (you can use "null !== expression" instead) in Command line code on line 1
```

Comment (nanaya) [inline app/Models/Forum/Topic.php]: looks like the artisan tinker shell ate the message.

```
$ php --version
PHP 7.0.8 (cli) (built: Jul  7 2016 01:16:14) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.8, Copyright (c) 1999-2016, by Zend Technologies
$ php -r "isset(null);"
PHP Fatal error:  Cannot use isset() on the result of an expression (you can use "null !== expression" instead) in Command line code on line 1

Fatal error: Cannot use isset() on the result of an expression (you can use "null !== expression" instead) in Command line code on line 1
$ artisan tinker
Psy Shell v0.7.2 (PHP 7.0.8 — cli) by Justin Hileman
>>> isset(null)
PHP Parse error: Syntax error, unexpected ')', expecting '[' on line 1
>>>
```

**Pull request merged**